src/PluginRegistrationDialog.cpp

```diff
@@ -34,6 +34,8 @@
 #if wxUSE_ACCESSIBILITY
 #include "widgets/WindowAccessible.h"
 
+#define DISABLE_STATE_NEW
```

Comment:
The constants `STATE_New` and `ID_ShowNew` might also exist only when conditionally compiled.  That would make it more obvious that the changes in this commit are complete and correct.

That would mean one of the event bindings in the table is also eliminated, and one case of a switch statement, and one if-block in `SetState`.